specs/guidelines.md

```diff
@@ -130,3 +140,51 @@ In case of doubt check other existing models!
 + `source`
 + `relativeHumidity`
 + `temperature`
+
+## <a name="contribute"></a>How to contribute
```

Comment:
Remove the anchor tag - Readthedocs sometimes can't cope with `<a>` and it screws up the formatting